custom_components/pyscript/state.py

```diff
@@ -98,7 +100,7 @@ def notify_var_get(cls, var_names, new_vars):
         return notify_vars
 
     @classmethod
-    def set(cls, var_name, value, new_attributes=None, **kwargs):
+    async def set(cls, var_name, value, new_attributes=None, **kwargs):
```

Comment:
Does this need to be async?

Reply:
The `RestoreStateData.async_get_instance` method is async, so I think so? Or is there a better way to do this?

Reply:
Oh sorry - I missed that `await`.  Yes, making the `set` function async is fine.